metals/src/main/scala/scala/meta/internal/metals/ammonite/Ammonite.scala

```diff
@@ -263,11 +263,6 @@ final class Ammonite(
           _ <- connectToNewBuildServer(conn)
         } yield ()
       }
-      .recoverWith {
```

Comment:
This would mean that two messages showed up, one about an error and another that the build import suceeded, which it didn't. `Future(())` was the cause of that.